src/app/shared/components/product/components/product-list/product-list.component.html

```diff
@@ -0,0 +1,12 @@
+@if (vm(); as vm) {
+  <div class="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-8 mt-10 transition-all">
+    @for (product of vm.productItems; track $index) {
+      <app-product-item [routerLink]="[vm.link, $index]">
+        <p slot="id">0{{ $index + 1 }}</p>
```

Comment:
What is this ID? It does not show up in the VM, but it has a slot defined, it seems a bit funky...